### ISSUE TITLE
Image size in crop view not fullscreen

### ISSUE THREAD

**Issue by Macbiont** [AlexLittlejohn/ALCameraViewController#260]
Since the last updates the image is scaled 50% and does not use the full screen width | height anymore.

![img_b902e86528d3-1](https://user-images.githubusercontent.com/2786054/37463704-5d4c39a0-2856-11e8-98f7-5d3b7d8caac8.jpeg)

Any ideas to solve that?

**Comment by jngr** [comment]
I agree that this is not optimal. While before (<= v. 3.02, some part of the image could not be included as the image always covered the entire screen width, now the user could select a non-square area, which is then stretched (i.e. distorted) to square format.

Ideal would be what UIImagePickerController does with landscape format, i.e. zoom the image min. to fill the entire square (in portrait format, it unfortunately adds black bars left and right).

It is a bit strange as the 3.0.3 release included a fix for issue #244 describing this problem, which I have not seen with my fixed square cropping area until now.

**Comment by Macbiont** [comment]
@jngr did you fix that issue? I'm unsure if ALCameraViewController will be developed further.

**Comment by jngr** [comment]
@Macbiont Sorry - version 3.0.2 works fairly well for my purposes, so I haven't looked into this any further.

**Comment by dean20160218** [comment]
hi how to solve?

**Comment by rtrsasikumar** [comment]
Hi, how to solve?
